src/Database/Database.php

```diff
@@ -590,7 +599,7 @@ public function dropTable($table): bool
         }
 
         // update cache
-        $key = array_search($this->tableWhitelist, $table);
+        $key = array_search($table, $this->tableWhitelist ?? []);
```

Comment:
Is this a bugfix? If so, it really should have its own PR.
We need to keep actual changes to the code separate, so it's easier to review them and see if it's a breaking change.

Reply:
An overlooked bugfix, yes.

Reply:
@distantnative So, i can't test methods when i revert back.  I don't think there will be a break change because it's a critical error. It wasn't working already 😅 

> array_search() expects parameter 2 to be array, string given

Reply:
Separated https://github.com/getkirby/kirby/pull/2788